MainModule/Server/Core/Functions.lua

```diff
@@ -895,12 +895,12 @@ return function(Vargs, GetEnv)
 		Message = function(sender, title, message, image, players, scroll, duration)
 
 			-- Currently not used
-			if sender == 'Adonis' or sender == 'HelpSystem' or sender == 'Command' then
+			if sender == "Adonis" or sender == "HelpSystem" or sender == "Command" then
 				sender = nil
 			end
 
 			-- ////////// Compatability for older plugins (before sender and image ares were introduced)
-			if sender ~= nil and typeof(sender) ~= 'Instance' and typeof(sender) ~= 'userdata' and typeof(sender) ~= 'table' then
+			if sender ~= nil and type(sender) ~= "Instance" and type(sender) ~= "userdata" and type(sender) ~= "table" then
```

Comment:
This wont work, you cant get "Instance" from type(), it will return "userdata", you may as well just remove the first type guard.

Reply:
Oh that one was a mistake